airflow/sensors/smart_sensor.py

```diff
@@ -341,21 +340,21 @@ def _load_sensor_works(self, session=None):
                 .filter(SI.state == State.SENSING)
                 .filter(SI.shardcode < self.shard_max, SI.shardcode >= self.shard_min)
             )
-            tis = query.all()
+            sis = query.all()
 
-        self.log.info("Performance query %s tis, time: %.3f", len(tis), timer.duration)
+        self.log.info("Performance query %s sis, time: %.3f", len(sis), timer.duration)
 
         # Query without checking dagrun state might keep some failed dag_run tasks alive.
         # Join with DagRun table will be very slow based on the number of sensor tasks we
         # need to handle. We query all smart tasks in this operator
         # and expect scheduler correct the states in _change_state_for_tis_without_dagrun()
 
         sensor_works = []
-        for ti in tis:
+        for si in sis:
             try:
-                sensor_works.append(SensorWork(ti))
+                sensor_works.append(SensorWork(si))
             except Exception:
-                self.log.exception("Exception at creating sensor work for ti %s", ti.key)
+                self.log.exception("Exception at creating sensor work for ti %s", si.ti_key)
```

Comment:
Does `SensorInstance` have a `ti_key` attribute?

Reply:
(I couldn't find `si.key` too)

Reply:
It does, yes.

Reply:
Er, no. Reverted this change. (I was thinking wrong class)